src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,11 @@
+body,
+div {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
move this property to `*` selector

Reply:
We were taught in lections not to use this selector at all.

Reply:
It is ok for `box-sizing`